examples/pytorch/README.md

```diff
@@ -252,3 +252,65 @@ or if in a Conda environment:
 ```bash
 conda install -c comet_ml -c anaconda -c conda-forge comet_ml
 ```
+
+### Neptune
+
+1. Install the Neptune client library:
+
+    pip:
+
+    ```bash
+    pip install neptune-client
+    ```
+
+    conda:
+
+    ```bash
+    conda install -c conda-forge neptune-client
+    ```
+
+1. Import `NeptuneCallback`:
+
+    ```python
+    from transformers.integrations import NeptuneCallback
+    ```
+
+1. Enable Neptune logging in your script:
+
+    - Create a Neptune callback and pass it to the Trainer:
+
+        ```python
+        callback = NeptuneCallback()
+        trainer = Trainer(
+            model,
+            ...
+            callbacks=[callback],
+        )
+        ```
+
+    - Alternatively, in your `TrainingArguments`, set the `report_to` argument to `"neptune"`:
```

Comment:
In HF, report_to is default, user who needs more customization goes with callbacks.